boards/arm/hexiwear_k64/hexiwear_k64.dts

```diff
@@ -17,6 +17,9 @@
 		red-pwm-led = &red_pwm_led;
 		green-pwm-led = &green_pwm_led;
 		blue-pwm-led = &blue_pwm_led;
+		gpio-a = &gpioa;
+		gpio-b = &gpiob;
+		gpio-c = &gpioc;
```

Comment:
Is that needed?

Reply:
Its is until we have nodelabel generation support.  See commit message for `boards: hexiwear_k64: add alias until node labels exist`